distributed/distributed-schema.yaml

```diff
@@ -855,6 +855,6 @@ properties:
           and compiled with hwloc support. Unexpected errors can occur when using
           ``"auto"`` if any interfaces are disconnected or improperly configured.
       reuse-endpoints:
-        type: boolean
-        description: | 
```

Comment:
I think it would be helpful to include the language in this PR (at the top) for the description.

Reply:
Good idea, done in https://github.com/dask/distributed/pull/4836/commits/7d5db9c5ae241079d23aba64816e2f9a120e268f .